test/net_http_test.rb

```diff
@@ -105,6 +105,49 @@ def test_get_is_protected
     end
   end
 
+  def test_get_with_no_bulkhead
```

Comment:
this tests that a success response happens if the ticket count is `false` - given the tickets get passed to the resource - which was crashing because not initialized before hand. Was trying not to initialize semaphore and skip acquiring it.